<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QMainWindow" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,1">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="player_label">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Player #</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="PlaceShips" name="place_ships" native="true"/>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="wait_label">
      <property name="text">
       <string>Waiting for opponent...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="fight" name="player_fight" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaceShips</class>
   <extends>QWidget</extends>
   <header>placeships.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>fight</class>
   <extends>QWidget</extends>
   <header>fight.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
